pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  nodejs,
+  pnpm_9,
+  autoPatchelfHook,
+  llvmPackages,
+  musl,
+  xorg,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wrangler";
+  version = "3.62.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "workers-sdk";
+    rev = "wrangler@${finalAttrs.version}";
+    hash = "sha256-/4iIkvSn85fkRggmIha2kRlW0MEwvzy0ZAmIb8+LpZQ=";
+  };
+
+  pnpmDeps = pnpm_9.fetchDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-aTTaiGXm1WYwmy+ljUC9yO3qtvN20SA+24T83dWYrI0=";
+  };
+
+  buildInputs =
+    # cpp is required for building workerd.
+    # workerd is used for some wrangler subcommands.
+    # non-linux platforms may experience errors
+    # on certain subcommands due to this issue.
+    [
+      llvmPackages.libcxx
+      llvmPackages.libunwind
+      musl
+      xorg.libX11
+    ];
```

Comment:
Please format this again with `nixfmt-rfc-style`.

Reply:
i already used nixfmt. i believe it's the rfc-style already.
just to be sure, i used the instructions from nixfmt github
`nix-env -f https://github.com/NixOS/nixfmt/archive/master.tar.gz -i`
and installed an entirely different copy. it had the same results.

Reply:
Hmm I'm pretty sure that after after you'll move that comment nixfmt will format it. The best way to run it though is to use `direnv` (with a recent enough checkout that includes a `.envrc` file).